main.go

```diff
@@ -99,7 +99,7 @@ func handleEvent(eventType string, bytesIn []byte) error {
 		}
 		break
 	default:
-		return fmt.Errorf("X_Github_Event want: ['pull_request', 'issue_comment'], got: " + eventType)
+		fmt.Printf("X_Github_Event was uninteresting want: ['pull_request', 'issue_comment'], got: " + eventType)
```

Comment:
I think `was uninteresting` is a bit too conversational. 

We did return an error but now we print to the console.. how come?

Reply:
The returning of an error results in a http/500 which suggests a state or condition as being wrong.  The application has actually behaved as intended but the received event was not of interest to it.

Uninteresting is not opinion, its descriptive.  Short for "the received event was not in the list of event types that Derek currently parses".